packages/core/src/lib/toast/toast-banner.svelte

```diff
@@ -20,11 +20,11 @@ export { extraClasses as cx };
 
 <div
   class={cx(
-    'relative flex w-auto items-center gap-2 border bg-medium p-2 text-sm',
+    'relative flex h-10 w-auto max-w-[480px] items-center border bg-medium pl-3 pr-1 text-sm',
```

Comment:
Use `w-max` rather than `w-auto` to set `width: max-content` on the items. This ensures they display correctly (sized to their contents) in the storybook

```suggestion
    'relative flex h-10 w-max max-w-[480px] items-center border bg-medium pl-3 pr-1 text-sm',
```

Current:
![Screenshot 2024-01-09 at 12 52 08](https://github.com/viamrobotics/prime/assets/2963448/e4d50b7c-c57f-43ee-9475-078c05cb0d4b)

Expected:
![Screenshot 2024-01-09 at 12 52 13](https://github.com/viamrobotics/prime/assets/2963448/569e38d3-8c52-4513-8db1-01b97c9d0d3c)